llvm/test/MC/ELF/cfi-sframe-cfi-escape-errors.s

```diff
@@ -0,0 +1,37 @@
+# RUN: llvm-mc --filetype=obj --gsframe -triple x86_64 %s -o %t.o 2>&1 | FileCheck %s
+# RUN: llvm-readelf --sframe %t.o | FileCheck %s --check-prefix=NOFDES
+
+# Tests that .cfi_escape sequences that are unrepresentable in sframe warn
+# and do not produce FDEs.
+
+        .align 1024
```

Comment:
The file mixes tabs and spaces. Should standardize the style. In newer tests we use `## ` for comments that are not RUN or CHECK.